kordac/regex-list.json

```diff
@@ -30,5 +26,9 @@
     },
     "button-link": {
         "pattern": "\\{button (?P<args>[^\\}]*)\\}"
+    },
+    "text-box": {
```

Comment:
This snuck in somehow, will not have significant affects on the repository integrity. Will remove if requested.